src/Modules/Backend/templates/base/formTheme.html.twig

```diff
@@ -642,3 +642,10 @@
     {{ form_widget(form.children|last) }}
   </div>
 {% endblock %}
+
+{% block two_factor_authentication_widget %}
+  {{ form_widget(form) }}
+  {% if form.parent.vars.value.qrCode %}
+    <img src="{{ form.parent.vars.value.qrCode }}" class="img-fluid" alt="QR Code">
```

Comment:
change the alt to a translation
`Use th QR code to configure your authenticator app.`

Reply:
fixed 5b995b91ede947614eb480c3a83337354604c688